app/src/main/java/io/apicurio/registry/mt/TenantContextLoader.java

```diff
@@ -70,9 +71,11 @@ public class TenantContextLoader {
 
     @Inject
     @ConfigProperty(defaultValue = "60000", name = "registry.tenants.context.cache.check-period")
+    @Info( category = "mt", description = "Tenants context cache check period", availableSince = "2.1.0.Final")
     Long cacheCheckPeriod;
 
     @ConfigProperty(name = "registry.organization-id.claim-name")
+    @Info( category = "mt", description = "Organization Id claim name", availableSince = "2.1.0.Final")
     List<String> organizationIdClaims;
 
```

Comment:
Suggest uppercase `ID` in description (also applies to `quarkus.oidc.client-id`)